Prototope/Behaviors.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  Collisions.swift
+//  Prototope
+//
+//  Created by Saniul Ahmed on 07/02/2015.
+//  Copyright (c) 2015 Khan Academy. All rights reserved.
+//
+
+import Foundation
+
+//MARK: Public Behavior Types
+
+// Protocol for types describing the "configuration" of a behavior.
```

Comment:
Mind using docstring syntax for these and the other comments here so they show up in Xcode autocomplete?